package-overrides/github/twbs/bootstrap@3.3.7.json

```diff
@@ -0,0 +1,24 @@
+{
+  "main": "js/bootstrap",
+  "shim": {
+    "js/bootstrap": {
+      "deps": ["jquery"],
+      "exports": "$"
+    }
+  },
+  "dependencies": {
+    "jquery": ">=1.9.1 <4"
```

Comment:
This version format is not supported in jspm - we only support semver ranges. Perhaps try just `jquery: '*'` here?